website/docs/reference/exposure-properties.md

```diff
@@ -55,3 +40,52 @@ exposures:
   - name: ... # declare properties of additional exposures
 ```
 </File>
+
+
+## Example
+
+<File name='models/jaffle/exposures.yml'>
+
+```yaml
+exposures:
+  
+  - name: weekly_jaffle_metrics
+    type: dashboard                         # required
```

Comment:
I think for "type", and "maturity" in particular, it's useful to have the list of available options in the reference section. Happy to just do this via a YAML comment for now :)

Reply:
Never mind, I see that it is above — resolving this comment